### PR TITLE
Improvement: Add initial GitHub Issue/PR templates

### PR DESCRIPTION
**Pull request by obbardc**
### What
This PR implements GitHub templates for new issues/PRs.

### Why
This is so I don't annoy @dalathegreat when opening PRs which do not follow the standard format ;-).

### How
Following github documentation to create the templates.

PS: Feel free to improve by editing the commit or commenting here. I will collect all changes.

### CODE REVIEW

**Comment by obbardc** [comment]
I thought about having separate templates for bug/feature request too, which could add labels to the resultant issue.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### How
+How does it do it?
+
+
+### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] Code is formatted correctly
+- [ ] CI tests pass
```
Comment (lenvm) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: This information is already included in the PR. Having it also in the PR description is duplicating the information, and adds redundant work for the PR creator.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+<!-- _Please make sure to review and check all of these items:_ -->
+
+
+### What
+This PR implements ...
+
+### Why
+Why does it do it?
+
+
```
Comment (dalathegreat) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: Remove this line to make it symmetric

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+### Expected Behavior
```
Comment (dalathegreat) [inline .github/ISSUE_TEMPLATE.md:1]: Like @amarofarinha suggested, we should add one field for Settings

### Settings
    Software Version
    Battery type
    Inverter communication protocol
    Hardware used for Battery-Emulator (LilyGo, Stark CMR, Custom)
    CONTACTOR_CONTROL (yes/no)
    DUAL_CAN (yes/no)
    WEBSERVER (yes/no)
    MQTT (yes/no)

**Review by dalathegreat — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (obbardc) [reply .github/PULL_REQUEST_TEMPLATE.md]: Actually, I removed it. Can always be added later if it helps.

**Review by dalathegreat — APPROVED** [review]

**Pull request merged**